provision/vipmanager.bash

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+declare -a ETCD
+
+while getopts "n:v:e:i:" option; do
+    case "${option}" in
+        n) NODENAME="${OPTARG}" ;;
+	i) VIP_ADDR="${OPTARG}" ;;
+        v) VIP_MANAGER_VERSION="${OPTARG}" ;;
+        e) ETCD[${#ETCD[@]}]=$OPTARG ;;
+    esac
+done
+
+if rpm -q vip-manager &>/dev/null ; then
+    echo "vipmanager was already installed"
+else
+    echo "Installing vipmanager"
+    rpm --import "/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7"
```

Comment:
This is useless. Eg.: 
https://github.com/ClusterLabs/PAF/blob/master/extra/vagrant/3nodes-vip/provision/system.bash#L35

Reply:
removed